### ISSUE TITLE
singleFile option for EnumValues

### ISSUE THREAD

**Issue by veeramarni** [dotansimha/graphql-code-generator#2164]
**Describe the bug**
 
When using `singleFile` options for Enum Values, we get below generated `enums`.

Also, document missing about this topic #2038 . 
```
import { __TypeKind } from "./enum";
import { __DirectiveLocation } from "./enum";
```
*To Reproduce**

3. My `codegen.yml` config file:

```yml
# Put your YML here
  packages/sample-client-state/src/interfaces/generated-models.ts:
    documents: "packages/sample-client-state/src/**/*.gql"
    schema: 
      - ./packages/sample-client-state/src/**/*.graphql
    config:
      enumValues: ./enum
      constEnums: true
```

**Expected behavior**

Should not generate them as they are not defined by the user.

**Environment:**

- `@graphql-codegen/...`:  1.4.0

**Additional context**

**Comment by dotansimha** [comment]
You are right, @veeramarni . We shouldn't apply any import/`enumValues` logic for the internal introspection types.

**Comment by dotansimha** [comment]
@veeramarni I think i fixed it in: https://github.com/dotansimha/graphql-code-generator/pull/2181 , can you please try it? (alpha is available `1.4.1-alpha-9f944327.7`)

**Comment by dotansimha** [comment]
Available in 1.5.0 🎉
